utils/mariadb/files/mysqld.init

```diff
@@ -134,13 +133,13 @@ start_service() {
 	fi
 
 	# Make sure all required directories exists and have correct rights
-	for i in "$logdir" "$rundir" "$sockdir"; do
+	for i in "$logdir" "$sockdir"; do
 		opts="-m 0750"
 		if ! [ -e "$i" ]; then
 			# $rundir needs to be accessible for
 			# clients
-			if [ "$i" = "$rundir" ]; then
-				opts=
+			if [ "$i" = "$sockdir" ]; then
```

Comment:
maybe it'd be better to make it a one liner"

```
[ "$i" = "$sockdir" ] && opts="-m 0755"
```

Reply:
The comment probably needs adjusting too.

Reply:
Done